cli/types/cypress.d.ts

```diff
@@ -5097,16 +5097,6 @@ declare namespace Cypress {
      * @see https://on.cypress.io/assertions
      */
     (chainer: 'not.match', value: string): Chainable<Subject>
-
-    // fallback
-    /**
-     * Create an assertion. Assertions are automatically retried until they pass or time out.
-     * Ctrl+Space will invoke auto-complete in most editors.
-     * @see https://on.cypress.io/should
-     */
-    (chainers: string, value?: any): Chainable<Subject>
-    (chainers: string, value: any, match: any): Chainable<Subject>
-
```

Comment:
This is going to be a pretty big breaking change, and I would argue that it's not a change in the positive direction. Consider that users can add custom Chai assertions, and that it's technically possible to rephrase chainers in many ways because of `.to`, `.a`, etc.

Do you see another way to accomplish this PR, without removing this fallback case? If not, we may not be able to merge this, since it seems to taketh away as much as it giveth.

Reply:
I suspected this might be the case.

There's more complex typescript definitions that might make it possible to compute all the possible strings, but these are usually super slow, and wouldn't include the second argument.

Therefore, unless anyone else sees this and has other ideas, I think this can be closed.